fuzzers/README.md

```diff
@@ -5,22 +5,21 @@
 You can find a large amount of example fuzzers built on top of LibAFL.
 They are sorted by focus:
 
-- `baby`: Minimal fuzzers and fuzzers demonstrating specific features that don't fit anywhere else.
-- `inprocess`: Common In-process fuzzers. Most of the time, this is what you want.
-- `binary_only`: Fuzzers for binary-only targets.
-- `forkserver`: Fuzzers that use a forkserver-style executor.
-- `full_system`: Fuzzers for full-system targets (kernels, firmwares, etc...).
-- `fuzzbench`: Fuzzbench fuzzers.
-- `structure_aware`: Grammar fuzzers, fuzzers for certain languages, fuzzers with custom inputs, and more.
-- `fuzz-anything`: Fuzzers for advanced targets like WASM or python, and other fuzzers that can be used for anything.
+- [`baby`](./baby/): Minimal fuzzers and fuzzers demonstrating specific features that don't fit anywhere else.
+- [`binary_only`](./binary_only/): Fuzzers for binary-only targets.
+- [`forkserver`](./forkserver/): Fuzzers that use a forkserver-style executor.
+- [`full_system`](./full_system/): Fuzzers for full-system targets (kernels, firmwares, etc...).
+- [`fuzz-anything`](./fuzz_anything/): Fuzzers for advanced targets like WASM or python, and other fuzzers that can be used for anything.
+- [`inprocess`](./inprocess/): Common In-process fuzzers. Most of the time, this is what you want.
+- [`structure_aware`](./structure_aware/): Grammar fuzzers, fuzzers for certain languages, fuzzers with custom inputs, and more.
 
 (Some fuzzers may fit into multiple categories, in which case we sort them as it makes sense, for example `structure_aware > full_system > binary_only > the rest`)
 
 ## Fully-feature Fuzzers
 
 Some rather complete fuzzers worth looking at are:
 
-- [`Libfuzzer_Libpng_Launcher`](./inprocess//): That's what most people want to use: our InProcess fuzzer with a lot of features like ASAn on some cores, multi threading (a better libfuzzer).
+- [`Libfuzzer_Libpng_Launcher`](./inprocess/libfuzzer_libpng_launcher): That's what most people want to use: our InProcess fuzzer with a lot of features like ASAn on some cores, multi threading (a better libfuzzer).
```

Comment:
wait
do you really want people to use launcher first?
if they have problems with launcher, often the problem is hidden, due to restarting , forking, .. etc

Reply:
It's the most fully-featured fuzzer, of course

Reply:
We should add features to enable simplemgr for debugging